src/Particles/babylon.particleSystem.ts

```diff
@@ -1593,10 +1593,19 @@
                 particleSystem.preventAutoStart = parsedParticleSystem.preventAutoStart;
             }
 
-            particleSystem.minEmitBox = Vector3.FromArray(parsedParticleSystem.minEmitBox);
-            particleSystem.maxEmitBox = Vector3.FromArray(parsedParticleSystem.maxEmitBox);
-            particleSystem.direction1 = Vector3.FromArray(parsedParticleSystem.direction1);
-            particleSystem.direction2 = Vector3.FromArray(parsedParticleSystem.direction2);
+            if (parsedParticleSystem.minEmitBox) {

```

Comment:
this should not be necessary
Only removing the 4 lines should be required. The ParticleSystem._Parse will take care of back compat